tests/functions/void_parameter.phpt

```diff
@@ -0,0 +1,8 @@
+--TEST--
+void return type: not valid as a parameter type
+--FILE--
+<?php
+
+function foobar(void $a) {}
+--EXPECTF--
+Fatal error: void cannot be used as a parameter type in %s on line %d
```

Comment:
Well, not really, we could rewrite the tests to only check if there's exception now. If anybody has the time ;)